6/s2i/bin/run

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# S2I run script for the 'nodejs' image.
```

Comment:
Maybe one more nitpick - can this comment be added to other versions of the `run` script so that we have less differences between versions?

Reply:
Fixed.